include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -619,7 +619,8 @@ struct OrtApiBase {
    *   older than the version created with this header file.
    */
   const OrtApi*(ORT_API_CALL* GetApi)(uint32_t version)NO_EXCEPTION;
-  const char*(ORT_API_CALL* GetVersionString)(void)NO_EXCEPTION;  ///< Returns a null terminated string of the version of the Onnxruntime library (eg: "1.8.1")
+  const char*(ORT_API_CALL* GetVersionString)(void)NO_EXCEPTION;    ///< Returns a null terminated string of the version of the Onnxruntime library (eg: "1.8.1")
```

Comment:
In general we should use const ORTCHAR_T* instead of const char*, because on Windows usually we prefer to use wide char.

Reply:
Good point. `GetVersionString` is originally defined using `char *`. I will change both.